internal/report/github/summary.go

```diff
@@ -0,0 +1,127 @@
+package github
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/saucelabs/saucectl/internal/report"
+)
+
+type Reporter struct {
```

Comment:
I think it'd be good to add a description of the purpose of this reporter (perhaps [a link as well](https://github.blog/2022-05-09-supercharging-github-actions-with-job-summaries/)?). It's not 100% clear otherwise (for a new developer that sees this code for the very first time).

Reply:
I added that precision !